index.html

```diff
@@ -1,338 +1,257 @@
+        respectful interaction on the internet is predicated on the assumption 
+        that we think of ourselves as an influential part of this digital community. 
+    </p>
+    <ul>
+        <li>
+            Closed communities, such as paid services or social media interest groups, 
+            should make every effort to 
+            <a target = none href = "https://blog.vanillaforums.com/community/community-guidelines-why-theyre-vital-and-how-to-improve-them">
+                create their own community guidelines.</a>
+        </li>
+        <li>
+            Given the complicated relationship between freedom of expression and the spread
+            of misinformaton, we believe that the Miller Test provides a useful community-oriented framework for 
+            determining what online content should be regulated by 
+            <a class = "listToggle">
+                asking...
+            </a>
+            <div class = "hiddenList">
+                <ul type = "square">
+                    <li>
+                        Whether the average person applying contemporary community 
+                        standards would find that the work, taken as a whole, appeals to the prurient interest.
+                    </li>
+                    <li>
+                        Whether the work depicts or describes, in a patently 
+                        offensive way, sexual conduct that would generate
+                        profound discomfort in minors and their guardiands.
+                    </li>
+                    <li>
+                        Whether digital interactions should be aimed at literary, artistic, 
+                        political, scientifical, or social pursuits.
+                    </li>
+                </ul>
+            </div>
+        </li>
+        <li>
+            Based on these questions, digital interactions should follow three values: 
+            <ul>
+                <li>
+                    They should provide a positive benefit to the internet, as determined by its community's needs/values.
+                </li>
+                <li>
+                    They should only violate governmentally-established laws when such laws prohibit them from
+                    offering the aforementioned positive benefits to the internet. 
+                </li>
+                <li>
+                    They should be aimed at literary, artistic, political, scientifical, or social pursuits.
+                </li>
+            </ul>
+        </li>
+    </ul>
+</article>
+
+<article onclick = "toggleSources()" id = "sourceInfo">
+    <p>
+        <br \>
+        Sources for our project
+        <br \>
+    </p>
+</article>
+
+<article id = "sources">
+    <ul>
+        <a target = none href = "https://www.oif.ala.org/oif/the-universal-declaration-of-human-rights-article-19-at-seventy/"> https://www.oif.ala.org/oif/the-universal-declaration-of-human-rights-article-19-at-seventy/</a>  <br />

```

Comment:
if u want the link to open in a new tab the proper syntax is `target="_blank"` not `target = none`, this might work, but only b/c some browsers try to "fix" mistakes or make assumptions, but it's best to to use the proper syntax to avoid potential bugs on other browsers u might not have tested with